src/main/drivers/accgyro/gyro_sync.c

```diff
@@ -73,7 +74,19 @@ void gyroSetSampleRate(gyroDev_t *gyro)
             gyro->gyroRateKHz = GYRO_RATE_2000_Hz;
             gyroSampleRateHz = 2000;
             accSampleRateHz = 800;
-            break;
+            switch(gyroConfig()->gyro_hardware_lpf) {
```

Comment:
It's a good programming practise to handle all enum values in a switch-case.

Reply:
done!